src/stylesheets/elements/form-controls/_checkbox-and-radio.scss

```diff
@@ -115,3 +115,26 @@
   box-shadow: 0 0 0 units($theme-input-select-border-width) color("disabled");
   cursor: not-allowed;
 }
+
+// Checkboxes and radios with tap-friendly targets
+.usa-checkbox__input--tile + .usa-checkbox__label,
+.usa-radio__input--tile + .usa-radio__label {
+  border: units($theme-input-tile-border-width) solid color($theme-input-tile-border-color);
+  border-radius: radius($theme-input-tile-border-radius);
+  margin: units(1) 0;
+  padding: units(1.5) units(2) units(1.5) units(5);
+}
+
+.usa-checkbox__input--tile:checked + .usa-checkbox__label,
+.usa-radio__input--tile:checked + .usa-radio__label {
+  background-color: color($theme-input-tile-background-color-selected);
+  border-color: color($theme-input-tile-border-color-selected);
+}
+
+.usa-checkbox__label-description,
+.usa-radio__label-description {
```

Comment:
Do we want to have a lower specificity for these? That would allow them to be used in the default radios/checkboxes… but if used there, might not look right without a container.

Reply:
This level of specificity looks ok to me. Are you suggesting adding something like `.usa-checkbox__label--tile`? I don't think that's necessary

Reply:
No - I was wondering if it should be `..usa-checkbox__input--tile .usa-checkbox__label-description` which would only apply the styling inside the tile.